### PR TITLE
Products Page

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Added products page .
It consists of :

-  Header

-  sidebar which have filter in it 

- hero-section

### CODE REVIEW

**Comment by bhargaviRdy** [comment]
image
the text in the filters part became too small after a breakpoint. Have alook if you missed it

**Comment by Priya-Patel31** [comment]
> image the text in the filters part became too small after a breakpoint. Have alook if you missed it

Thank you for reminding me out . Now have a look as I have made changes in it.

**Review by JantuDeb — APPROVED** [review]
Clean and well-structured code.

File: products/products.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+        </div>
+        <!-- ******************navbar right ended**************** -->
+      </nav>
+      <!-- 
+      ********************************** Navbar ended ************************************ -->
+
+      <!-- **********************************Main section started ****************************** -->
+
+      <div class="main-content-container">
+        <div class="sidebar">
```
Comment (rkapoor10) [inline products/products.html:97]: you could have used aside here

**Review by rkapoor10 — APPROVED** [review]
looks good to merge

**Pull request merged**